docker/README.md

```diff
@@ -458,3 +458,60 @@ Below are instructions for starting a debug session in PhpStorm that will listen
 1. Back in the main configuration window, click 'Apply' then 'Ok'.
 
 1. You can now start a debug session by clicking 'Run -> Debug' in the main menu
+
+#### Remote Debugging with VSCode
+
+You'll need:
+
+- PHP Debug plugin installed in VSCode
```

Comment:
Do you think we add a link to the "correct" plugin? I see 2 "PHP Debug" extensions available, adding a link should help figure out which one to install

 ```suggestion
- [PHP Debug](https://github.com/felixfbecker/vscode-php-debug) plugin installed in VSCode
```